### PR TITLE
Menu item changes 8.5

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

**Comment by A-Ovchinnikov-mx** [comment]
This screenshot is wrong:
content/howto/collaboration-requirements-management/attachments/18448637/18580419.png

It shows a "location" breadcrumb, but it's gone from the next Changes pane. Photoshop! :)

**Comment by A-Ovchinnikov-mx** [comment]
Don't see changes on the following screenshot:
content/howto/data-models/attachments/18448743/18582158.png

Was it supposed to change?

(Sorry, cannot comment on files unfortunately)

File: content/refguide/collaborative-development.md [hunk 1]
```diff
@@ -117,11 +117,31 @@ The development line for Studio has been selected.
...
+
+If you disable Studio for a development line it was enabled for and do not enable it for any other development line, you will not be able to use collaborative development.
+
+{{% /alert %}}
+
+To disable Studio, do the following:
+
+1. Select the branch that is enabled for Studio.
+
+2. Click the **Disable doe Mendix Studio** button:
```
Comment (A-Ovchinnikov-mx) [inline content/refguide/collaborative-development.md]: Disable for ...? :)

**Comment by A-Ovchinnikov-mx** [comment]
I don't see any changes on this screenshot:
content/howto/general/attachments/18448703/18581309.png

(Sorry, GitHub doesn't allow to comment on binary files apparently :( )

File: content/refguide/create-branch-line-dialog.md [hunk 2]
```diff
@@ -1,40 +1,60 @@
...
+Select from which tagged version you want to create a branch line. Every time you create a deployment archive a tag is created so that you can always refer back to that version of the project.
 
-## 5 Branch Line (for Branch Line)
+## 6 Branch Name
 
-Select from which branch line you want to create another branch line. We recommend that you make branch lines only from the main line but in some cases branching a branch line can be very useful.
+Enter a name of the new branch line. 
 
-## 6 Revision (for Branch Line)
+## 7 Existing Branch Line
```
Comment (A-Ovchinnikov-mx) [inline content/refguide/create-branch-line-dialog.md]: Line**s**

File: content/refguide/version-control-menu.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The **Download from Version Control Server** option downloads an app from the Team Server or another SVN server. This creates a local working copy of the app for development. For more information on what settings are displayed in the **Download from Version Control Server** dialog window, see [Download from Version Control Server](download-from-version-control-dialog).
+
+## 7 Upload to Version Control Server
+
+The **Upload to Version Control Server** option uploads a local app to a new or existing Team Server repository, or to another SVN server. This is only possible if the app is not yet version controlled. For more information on what settings are displayed in the **Upload to Version Control Server** dialog window, see [Upload to Version Control Server](upload-to-version-control-dialog).
+
+## 8 Manage Branch Lines
+
+The **Manage Branch Lines** option manages branch lines on the version control server that can be used to develop functionality separately from the main line. For more information on the Branch Line Manager and creating a new branch line, see [Branch Line Manager](branch-line-manager-dialog) and [Create Branch Line](create-branch-line-dialog).
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/refguide/version-control-menu.md]: Implemented

File: content/refguide/version-control-menu.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+The **Upload to Version Control Server** option uploads a local app to a new or existing Team Server repository, or to another SVN server. This is only possible if the app is not yet version controlled. For more information on what settings are displayed in the **Upload to Version Control Server** dialog window, see [Upload to Version Control Server](upload-to-version-control-dialog).
+
+## 8 Manage Branch Lines
+
+The **Manage Branch Lines** option manages branch lines on the version control server that can be used to develop functionality separately from the main line. For more information on the Branch Line Manager and creating a new branch line, see [Branch Line Manager](branch-line-manager-dialog) and [Create Branch Line](create-branch-line-dialog). 
+
+## 9 Merge Changes Here
+
+The **Merge Changes Here** option merges changes that were committed in another development line to the development line that is currently opened in Studio Pro.
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/refguide/version-control-menu.md]: Implemented

File: content/refguide/create-branch-line-dialog.md [hunk 5]
```diff
@@ -1,40 +1,60 @@
...
+## 4 Branch Line 
 
-Select from which revision of the main line you want to create a branch line. Often, you want to choose the most recent version. You then develop some functionality in the branch line and merge it back to the main line when you are done.
+This settings is only available when you select [Branch line](#branch-line) in **Create branch from**. 
+
+Select from which branch line you want to create another branch line. We recommend that you make branch lines only from the main line but in some cases branching a branch line can be useful.
+
+## 5 Tagged Version
+
+This settings is only available when you select [Tagged version](#tagged-version) in **Create branch from**.
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/refguide/create-branch-line-dialog.md]: Fixed

File: content/refguide/project-menu.md [hunk 6]
```diff
@@ -8,118 +8,72 @@ tags: ["Studio Pro", "project menu", "top bar"]
 
 ## 1 Introduction
 
-In the **Project** menu you can view and/or manipulate settings that are connected to the version control or deployment. For example, you can view the history of the current development line or create a deployment package.
+In the **Project** menu you can view and/or manipulate settings that are connected to the your project and deployment. For example, you create a deployment package.
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/refguide/project-menu.md]: Fixed

File: content/refguide/create-branch-line-dialog.md (hunk 2 above)
Comment (MariaShaposhnikova) [reply content/refguide/create-branch-line-dialog.md]: Fixed

**Comment by MariaShaposhnikova** [comment]
@A-Ovchinnikov-mx , thanks for the review, I have made changes including the screenshot mentioned above.

**Pull request merged**